gateway/setup/create_certs.sh

```diff
@@ -0,0 +1,19 @@
+if ! command -v openssl &> /dev/null
+then
+    echo "openssl could not be found"
+    exit
+fi
+mkdir -p ../certs
+cd ../certs
+# create CA private key
+openssl genrsa -des3 -out ca_key.pem 2048
```

Comment:
3DES is a deprecated encryption algorithm now - even though a lot of OpenSSL guidelines still have it via copy & paste. AES128 is the recommended cipher for most applications these days. So replace the `-des3` with `-aes128`.